app/helpers/papersQueryFormatter.ts

```diff
@@ -10,6 +10,7 @@ export interface FilterObject {
   journalIFTo?: number | string;
   fos?: number[] | string[];
   journal?: number[] | string[];
+  [key: string]: any;
```

Comment:
Please remove this line.
And change `isFilterEmpty` method to use `explicit any` type cast.